recipe/meta.yaml

```
@@ -33,16 +32,22 @@ requirements:
   host:
     - r-base
     - r-rcpp
+    # - bioconductor-aldex2
+    - r-corpcor
     - r-fastcluster
-    - r-ggplot2
-    - r-igraph
+    # - bioconductor-limma
+    - r-ppcor
+    - r-vegan
   run:
     - r-base
     - {{ native }}gcc-libs         # [win]
     - r-rcpp
+    # - bioconductor-aldex2
+    - r-corpcor
     - r-fastcluster
-    - r-ggplot2
-    - r-igraph
+    # - bioconductor-limma
+    - r-ppcor
+    - r-vegan
```

AFAICT, only `Rcpp` is actually a run dependency. Unlike CRAN, we don't install Suggests packages during build - these are usually needed for vignette rendering and running examples, which we omit.

I suggest this be simplified to:

```suggestion
  host:
    - r-base
    - r-rcpp
  run:
    - {{ native }}gcc-libs         # [win]
    - r-base
    - r-rcpp
```

---

I actually need to be able to use r-corpcor and r-ppcor during run time. Is it not possible to include them?